src/inspector_socket_server.cc

```diff
@@ -25,9 +25,18 @@ void Escape(std::string* string) {
 }
 
 std::string GetWsUrl(const std::string& host, int port, const std::string& id) {
-  char buf[1024];
-  snprintf(buf, sizeof(buf), "%s:%d/%s", host.c_str(), port, id.c_str());
-  return buf;
+  // Host is valid (socket was bound) so colon means it's a v6 IP address
```

Comment:
I found it. Moved the function to inspector_io.{h,cc}. Since the _server class does not depend on anything in the Node (so it is easier to test it) I copied a declaration to the inspector_socket_server.cc.